tests/remote/test_fennec.py

```diff
@@ -19,18 +19,7 @@
     # Support for API level 15 ended on Aug 29th 2017
     {'application': 'fennec', 'platform': 'android-api-15',
      'branch': 'mozilla-central', 'date': '2017-08-29'},
-    {'application': 'fennec', 'platform': 'android-api-16', 'branch': 'mozilla-central'},
     {'application': 'fennec', 'platform': 'android-x86', 'branch': 'mozilla-central'},
```

Comment:
Maybe you can change that line to make use of a date instead like https://archive.mozilla.org/pub/mobile/nightly/2019/05/2019-05-17-16-29-00-mozilla-central-android-x86/